src/twisted/mail/test/test_imap.py

```diff
@@ -3007,12 +3031,12 @@ def test_tooManyArgumentsForCommand(self):
         self.server.lineReceived(b"001 LOGIN A B C")
         self.assertEqual(
             self.transport.value(),
-            networkString(
-                "001 BAD Illegal syntax:"
-                " Too many arguments for command: %r"
-                "\r\n"
-                % (b"C",)
-            ))
+            (b"001 BAD Illegal syntax:" +
+             b" Too many arguments for command: " +
+             repr(b'C').encode("utf-8") +
```

Comment:
Yes, I think you are right.

Reply:
No, this needs to be like this because in imap4.py, there is this:

```
        if line:
            # Too many arguments
            raise IllegalClientResponse("Too many arguments for command: " + repr(line))
```

so the value of this is different on Python 2 and Python 3.
Maybe this can be cleaned up in a follow-up PR, but I don't want to fix that here.